lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -39,12 +39,7 @@
                     user: user,
                     enrollments: this.enrollments,
                     formatDate: function(date) {
-                        if (!date) {
-                            return 'N/A';
-                        }
-                        else {
-                            return moment(date).format('MM/DD/YYYY (H:MM UTC)');
-                        }
+                        return date ? moment.utc(date).format('lll z') : 'N/A';
```

Comment:
@sstack22  Yes you are right. @mushtaqak gave just an example. I have added course start date in Studio : 10/01/2013 and its means Oct 1, 2013. :)